settings.yml

```diff
@@ -1,11 +1,26 @@
+api_provider:
+  default: 'giphy'
+  type: 'enum'
+  description:
+    en: "Which gifs provider should we use"
+  choices:
+    - giphy
+    - tenor
 giphy_api_key:
   type: string
   default: ""
-  description: "The GIPHY API key"
+  description:
+    en: "GIPHY API key"
+tenor_api_key:
+  type: string
+  default: ""
+  description:
+    en: "TENOR API key"
 giphy_locale:
   type: enum
   default: en
-  description: "Language to use on the search. Used to cater search to regional content. Set to your default forum locale."
+  description:
+    en: "Language to use on the search. Used to cater search to regional content. Set to your default forum locale."
```

Comment:
```suggestion
    en: "GIPHY: Language to use for search results. Used to cater search to regional content. Set to your default forum locale."
```

Reply:
This is generic to both Tenor and Giphy so the description should remain generic.  The setting is only called giphy_locale to provide upgrade compatibility.